.github/workflows/release.yml

```diff
@@ -32,6 +32,32 @@ jobs:
         with:
           name: authn-linux64
           path: dist/authn-linux64
+  linux64Arm:
+    name: Compile for Linux ARM
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-go@v2
+        with:
+          go-version: ^1.17
+      - run: go mod download
+
+      - name: compile
+        run: |
+          go build \
+            -ldflags "-extldflags -static -X main.VERSION=${VERSION##*/v}" \
+            -o dist/authn-linux64arm
+        env:
+          VERSION: ${{ github.ref }}
+          GOOS: linux
+          GOARCH: arm64
+          CGO_ENABLED: 1
+
+      - uses: actions/upload-artifact@v2
+        with:
+          name: authn-linux64arm
+          path: dist/authn-linux64arm
```

Comment:
you might be able to verify the `linux64Arm` step by running it in your fork, grabbing the artifact, and making sure it executes as expected